force-app/main/educationCloudSettings/classes/settingsProduct/EDCSettingsProductAPIService.cls

```diff
@@ -93,7 +104,7 @@ public virtual with sharing class EDCSettingsProductAPIService {
         Callable settingsProductInformationAPIService;
         String nonEDANamespace = namespace;
 
-        if (namespace.equalsIgnoreCase('hed')) {
+        if (String.isNotBlank(namespace) && namespace.equalsIgnoreCase('hed')) {
```

Comment:
We are removing `hed` namespace here, will this work in an actual managed package context with `hed` namespace? Afaik scratch orgs don't have a namespace

Reply:
Thanks, just wanted to make sure it works in namespaced org
Maybe we can test in a namespaced scratch org - cci flow run dev_org_namespaced

Reply:
sounds good, last time I ran it using the 2gp and it worked, I'll also try it out in dev_org_namespaced

Reply:
@iskander-sfdc Just tested it in a namespaced_org:
![image](https://user-images.githubusercontent.com/63684581/128923535-a394351a-3488-4bf8-9d0f-3010f09582dc.png)

Reply:
You'll want to test in a 2gp org. I did earlier and it panned out.

Reply:
sure :-)